src/ecrc-frontend/src/components/page/transition/Transition.test.js

```diff
@@ -4,12 +4,22 @@ import { create } from "react-test-renderer";
 import Transition from "./Transition";
 
 describe("Transition Page Component", () => {
-  test("Matches the snapshot", () => {
-    const header = {
-      name: "Criminal Record Check"
-    };
+  const header = {
+    name: "Criminal Record Check"
+  };
 
-    const transition = create(<Transition header={header} />);
-    expect(transition.toJSON()).toMatchSnapshot();
+  const page = {
+    header
+  };
+
+  test("Matches the default snapshot", () => {
+    const defaultTransition = create(<Transition page={page} />);
+    expect(defaultTransition.toJSON()).toMatchSnapshot();
+  });
+  test("Matches the non-whitelisted snapshot", () => {
+    const nonWhitelistTransition = create(
+      <Transition page={{ ...page, transitionReason: "notwhitelisted" }} />
+    );
+    expect(nonWhitelistTransition.toJSON()).toMatchSnapshot();
```

Comment:
same here, a test case for bcsc

Reply:
Same here can change the name to reflect that default will display bcsc

Reply:
yup! would be great